airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -74,7 +74,7 @@ def __init__(
                 raise ValueError("Cannot specify num_of_dpus with custom WorkerType")
         elif not worker_type_exists and num_workers_exists:
             raise ValueError("Need to specify custom WorkerType when specifying NumberOfWorkers")
-        elif worker_type_exists and not num_workers_exists:
+        elif worker_type_exists:
```

Comment:
I might argue that while these are functionally the same, the old way was a little more clear about the intent.  Just a thought.